### DISCUSSION TITLE
Move without resizing?

### DISCUSSION THREAD

**Discussion by deandmx** [rxhanson/RectanglePro-Community#121] [Q&A]
Are there any actions to just move windows without having to resize them also? For example, move a window to the corners of the screen. Preferably while respecting my gaps between screen edges setting.

**Comment by rxhanson** [comment]
You can use actions in the "Snap" category for this:

![image](https://user-images.githubusercontent.com/13651296/192162678-d01af30a-d28a-4589-8d99-4569ac2a62d7.png)

Or you can create your own shortcuts, and just leave w & h empty. This way you can get corners with a single shortcut:

![image](https://user-images.githubusercontent.com/13651296/192162738-2302972a-42dd-4989-be11-7afc0f5b31ea.png)

**Comment by deandmx** [comment]
While this is useful for keyboard shortcuts I primarily use the app through Snap areas, mostly the Snap panel and edges & corners. I can't select custom shortcuts with no width or height as Snap areas for edges & corners or the Snap panel.

If you added corners to Snap that would solve my issue. This would allow them to be selected anywhere in the app without any custom configuration on the users end.

**Reply by rxhanson** [reply]
I see what you mean. I'll add this to my backlog for when I implement the "macOS center". 

Custom shortcuts can be added to the snap panel, but there is a bug with adding items to the snap panel that you might have hit, and right now custom shortcuts in the snap panel show up as the same icon (fixing both of these is on my todo for an upcoming release).

**Comment by deandmx** [comment]
Appreciate the feature additions in the latest update. One small extra request; snap doesn’t respect my screen border padding setting.

**Reply by rxhanson** [reply]
Ah, right. It'll be in the next release.
